lib/chef/chef_fs/file_system.rb

```diff
@@ -389,14 +391,17 @@ def copy_entries(src_entry, dest_entry, new_dest_parent, recurse_depth, options,
                       ui.output "Would update #{dest_path}" if ui
                     else
                       src_value = src_entry.read if src_value.nil?
+                      result["total"] += 1
                       dest_entry.write(src_value)
+                      result["success_count"] += 1
                       ui.output "Updated #{dest_path}" if ui
                     end
                   end
                 end
               end
             end
           rescue RubyFileError => e
+            # result["failed"].append(username)
```

Comment:
We should be capturing failures here and in the other exception cases, so that we can return failure information (file and error) as well.